src/_posts/languages/python/django/2000-01-01-start.md

```diff
@@ -262,15 +275,64 @@ command to execute in your postdeploy hook.
 The `migrate` command is the one to execute to apply to migrations to your
 database. You can add it to your `Procfile` like this:
 
+First create a file `bin/post_deploy.sh` with:
+
+```bash
+#!/bin/sh
+
+# Execute structure migrations
+python manage.py migrate users
+python manage.py migrate
+```
+
+After, activate the hook by adding following line in `Procfile`:
+
+```bash
+postdeploy: bash bin/post_deploy.sh
 ```
-web: gunicorn mydjango.wsgi --log-file -
-postdeploy: python manage.py migrate
+
+This will trigger a one-off container just after finishing compiling
+but before replacing current app (if any were running when triggering
+the deployment).
+
+{% note %}
+  More information in [postdeploy hook section]({% post_url platform/app/2000-01-01-postdeploy-hook %})
+{% endnote %}
+
+### Collect static
+
+Previous hook is not good for collect static because the script is
+run in a one-off which is a copy of the final container. Therefor all
```

Comment:
nitpick (non-blocking): from what I read, the syntax without the "e" is more formal. I find it excessive in the context of this documentation.

```suggestion
run in a one-off which is a copy of the final container. Therefore all
```